extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Utility functions for running Optimizely A/B tests with VisualEditor
+ */
+
+( function () {
+	/**
+	 * @returns {number} The A/B test variant number for the Toolbar A/B test
+	 */
+	ve.init.wikia.getToolbarABTestVariantNumber = function () {
```

Comment:
I think where the experiment is active depends on how you set up the URL filter in Optimizely.

The code in ve.init.wikia.Optimizely.js doesn't activate the experiment in any way, it just provides 
getToolbarABTestVariantNumber which is called on VE init from JS. 

With the alternative method you describe (setting a global variable to control the experiment) still wouldn't change this, the experiment will still run on any url that matches what is set in Optimizely. Or were you referring to some other way of controlling exactly when the experiment is activated programmatically?

I can probably get around this by adding &veaction=edit to the url filter, thus making sure it is only activated when the VE is loaded.